### PR TITLE
UIF-237 Display fiscal year rollover progress

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
Display fiscal year rollover progress
https://issues.folio.org/browse/UIF-237

## Approach
* load latest progress job and in case it's in progress status - re-fetch it every ten seconds by setInterval. 
* progress bar was imported from stripes-data-transfer-components

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    renderComponent({ ledgerName: LEDGER_NAME });
+    expect(screen.getByText(LEDGER_NAME)).toBeDefined();
+  });
+
+  it('should display progress bar', () => {
+    const { container } = renderComponent({
+      rolloverStatus: { budgetsClosingRolloverStatus: OVERALL_ROLLOVER_STATUS.inProgress },
+    });
+
+    expect(container.getElementsByClassName('progressCurrent')[0]).toHaveStyle('width: 33%');
```
Comment (axelhunn) [inline src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js:44]: why 33%? )

**Review by axelhunn — APPROVED** [review]

File: src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js (hunk 1 above)
Comment (aliaksei-chumakou) [reply src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js:44]: this is behavior of Progress component, kind of integration test)

Comment (axelhunn) [reply src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js:44]: I mean, it's not so obvious (for me as a new reader of this code), and some kind of magic

Comment (aliaksei-chumakou) [reply src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js]: changed to simple checking if it's displayed

File: src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { MemoryRouter } from 'react-router-dom';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
```
Comment (AndreiShumski2) [inline src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js:6]: is it required?

**Review by AndreiShumski2 — APPROVED** [review]

File: src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js]: nope

**Pull request merged**